cmd/generate/config/rules/aws.go

```diff
@@ -1,18 +1,22 @@
 package rules
 
 import (
-	"regexp"
-
+	"github.com/zricethezav/gitleaks/v8/cmd/generate/secrets"
 	"github.com/zricethezav/gitleaks/v8/config"
 )
 
-func AWS() *config.Rule {
+// https://summitroute.com/blog/2018/06/20/aws_security_credential_formats/
+
+var credFileAccessKey = "aws_access_key_id=AKIALALEMEL33243OLIB" // gitleaks:allow
+var credFileSecretKey = "aws_secret_access_key=" + secrets.NewSecret(hex("40"))
+var credFileSessionToken = "aws_session_token=" + secrets.NewSecret(hex("928"))
+
+func AWSAccessKey() *config.Rule {
 	// define rule
 	r := config.Rule{
 		Description: "Identified a pattern that may indicate AWS credentials, risking unauthorized cloud resource access and data breaches on AWS platforms.",
-		RuleID:      "aws-access-token",
-		Regex: regexp.MustCompile(
-			"(?:A3T[A-Z0-9]|AKIA|ASIA|ABIA|ACCA)[A-Z0-9]{16}"),
+		RuleID:      "aws-access-key",
```

Comment:
@baruchiro can we just leave this as `aws-access-token` so it doesn't break existing configs? We could add a little note in the description mentioning that this really should be called `aws-access-key`?

Reply:
I don't have an access to that branch anymore, sorry..

Here is my suggestion:

```go
Description: "Identified a pattern that may indicate AWS access key, risking unauthorized cloud resource access and data breaches on AWS platforms.",
RuleID:      "aws-access-token",
```